mne/utils.py

```diff
@@ -510,6 +510,21 @@ def _reject_data_segments(data, reject, flat, decim, info, tstep):
     return data, drop_inds
 
 
+def _get_inst_data(inst):
```

Comment:
sounds good

On Wed, Aug 3, 2016 at 11:13 AM, Mikolaj Magnuski <notifications@github.com>
wrote:

> In mne/utils.py
> <https://github.com/mne-tools/mne-python/pull/3479#discussion_r73387834>:
>
> > @@ -510,6 +510,21 @@ def _reject_data_segments(data, reject, flat, decim, info, tstep):
> >      return data, drop_inds
> >
> >
> > +def _get_inst_data(inst):
>
> I'll add support for other mne objects in a separate PR.
> Regarding copy param - I think in this case it is straightforward to just
> do _get_inst_data(inst).copy().
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3479/files/e326dac131c1919c4da572a00f9eb8debf3120e8#r73387834>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABwSHfTuG3zZVPNeWA9pp6dVqPEQ7ve6ks5qcNpBgaJpZM4JaFhW>
> .
>